database.py

```
@@ -0,0 +1,66 @@
+from sqlalchemy.exc import OperationalError
+import logging
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def get_db_connection():
+    """Establishes a connection to the database using the DATABASE_URL environment variable."""
+    db_url = os.getenv('DATABASE_URL')
+    if not db_url:
+        raise ValueError("DATABASE_URL environment variable is not set.")
```

Ruff (0.13.0)

Avoid specifying long messages outside the exception class

(TRY003)